programs/server/config.xml

```diff
@@ -409,12 +409,12 @@
     <!-- Approximate size of mark cache, used in tables of MergeTree family.
          In bytes. Cache is single for server. Memory is allocated only on demand.
          You should not lower this value.
+      <mark_cache_size>5368709120</mark_cache_size>
       -->
-    <mark_cache_size>5368709120</mark_cache_size>
 
     <!-- For marks of secondary indices.
+      <index_mark_cache_size>5368709120</index_mark_cache_size>
       -->
-    <index_mark_cache_size>5368709120</index_mark_cache_size>
 
     <!-- If you enable the `min_bytes_to_use_mmap_io` setting,
```

Comment:
For consistency, it would be cool if you could

- comment out `<mmap_cache_size>1000</mmap_cache_size>`

- comment out `<compiled_expression_cache_size>134217728</compiled_expression_cache_size>`

- comment out `<compiled_expression_cache_elements_size>10000</compiled_expression_cache_elements_size>`

- move

```xml
<query_cache>
    <max_size_in_bytes>1073741824</max_size_in_bytes>
    <max_entries>1024</max_entries>
    <max_entry_size_in_bytes>1048576</max_entry_size_in_bytes>
    <max_entry_size_in_rows>30000000</max_entry_size_in_rows>
</query_cache>
```

up, after `<compiled_expression_cache_elements_size>` (because it is yet-another-cache)

- then comment out each line of`<query_cache>` (as we don't want to give a wrong impression that the database default don't work out of the box)